Sources/KituraSession/SessionState.swift

```diff
@@ -76,6 +76,8 @@ public class SessionState {
             store.save(sessionId: id, data: data, callback: callback)
         } catch(let error as NSError) {
             callback(error)
+        } catch {
+            callback(nil)
```

Comment:
This seems contrary to the expected behavior (`callback(<non-nil>)`) if error is encountered. Is it possible to transform the caught error into an `NSError` object and feed it to the `callback`?